### DISCUSSION TITLE
799

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#974] [Comments]
# 799

전 직장에서는 문화 면접 혹은 수습 전환 인터뷰시에 창업자셨던 대표님이 참여하시면 꼭 물어보시는 질문이 있었다."좋은 회사란 무엇이라고 생각하시나요?"보상/복지가 좋은 회사, 가족들에게 자랑할 수 있는 회사, 뛰어난 동료가 모여있는 회사 등등 여러 좋은 회사들의 조건들이 있다.그 당시에는 되게 남들이 하는 그런 흔한 좋은 회사의 모습을 이야기했다.그때의 답변이 나 스스로도 너무 마음에 들지 않고, 한번도 고민해보지 못했던 주제라서 이따금 "좋은 회사" 에 대해 생각한다."좋은 회사" 라는게 결국 "계속 다니고 싶은 회사" 를 의미한다고 생각했다.그래서 2~3년 전부터 "좋은 회사란 같은 관심사와 미션을 가진 사람들이 모인 커뮤니티" 라고 나름 정리가 되었다.물론 회사의 제 1 목표는 회사의 지속된 생존이..

https://jojoldu.tistory.com/799

**Comment by bysxx** [comment]
항상 향로님의 글을 읽지만 오늘은 너무 공감되어 댓글을 남깁니다.

여태 다양한 회사 특히 스타트업들의 얘기를 들어봤을 때 계속 다니고 싶어 하고 행복을 느끼는 회사, 팀들의 공통점은 같은 관심사를 가지고 같은 고민을 하는 사람들이 모여있다는 점인 것 같습니다. (꼭 개발팀이 아니더라도)

그렇기에 모인 팀원들의 관심사가 비슷하다면 더할 나위가 없겠지만, 어떠한 상황에서라도 팀원들의 비슷한 관심사를 이끌어내고 개인의 고민이 아닌 팀의 고민이라는 느낌을 들게 해주는 게 리더의 중요한 역할이라고 생각합니다.

좋은 글 감사합니다.

**Comment by el-yun** [comment]
좋은 글 잘 읽었습니다. 저에게 좋은 회사는 제 페이스 대로 성과 낼 수 있고 보상이든 커리어든 동기부여가 잘되는 회사라고 언젠가부터 스스로 정의하고 있었는데 많이 공감하고 갑니다
